### PR TITLE
Projects container tests

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = {
   // collectCoverage: false,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  collectCoverageFrom: ['<rootDir>/src/**/*.js'],
+  collectCoverageFrom: ['<rootDir>/src/components/Projects/**/container.js'],
```
Comment (plzen) [inline jest.config.js]: Use all files to collect coverage

File: src/components/Projects/ProjectForm/__tests__/container.spec.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    instance.showManageButtons();
+    expect(instance.state.isShowManageButtons).toBeTruthy();
+  });
+
+  it('hideManageButtons()', () => {
+    instance.hideManageButtons();
+    expect(instance.state.isShowManageButtons).toBeFalsy();
+  });
+
+  it('handleReset()', () => {
```
Comment (plzen) [inline src/components/Projects/ProjectForm/__tests__/container.spec.js]: Add test for resetForm call

File: src/components/Projects/ProjectItem/ProjectItemForm/__tests__/container.spec.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    .dive();
+  const instance = container.instance();
+  const spyResetForm = jest.fn();
+
+  it('should match snapshot', () => {
+    expect(container).toMatchSnapshot();
+  });
+
+  it('handleReset()', () => {
+    container.setProps('resetForm', spyResetForm);
```
Comment (plzen) [inline src/components/Projects/ProjectItem/ProjectItemForm/__tests__/container.spec.js]: Try to use `jest.spyOn(container.props, 'resetForm')` instead of using `setProps`

File: src/components/Projects/ProjectItem/ProjectModalConfirmDelete/__tests__/container.spec.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  const container = wrapper.dive();
+  const instance = container.instance();
+
+  it('should match snapshot', () => {
+    expect(container).toMatchSnapshot();
+  });
+
+  it('handleDeleteProject()', () => {
+    instance.handleDeleteProject();
+    expect(deleteProject).toBeCalled();
```
Comment (plzen) [inline src/components/Projects/ProjectItem/ProjectModalConfirmDelete/__tests__/container.spec.js]: Don't need in case you have next line

File: src/components/Projects/ProjectItem/__tests__/container.spec.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  const wrapper = shallow(<ProjectItem {...props} store={store} />);
+  const container = wrapper.dive();
+  const instance = container.instance();
+
+  it('should match snapshot', () => {
+    expect(container).toMatchSnapshot();
+  });
+
+  it('handleRename()', () => {
+    expect(instance.state.isRename).toBeFalsy();
```
Comment (plzen) [inline src/components/Projects/ProjectItem/__tests__/container.spec.js]: Use setState before calling method

File: src/components/Projects/ProjectList/__tests__/container.spec.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    projects: [{}],
+    noItems: [{}],
+  };
+  const wrapper = shallow(<ProjectList {...props} store={store} />);
+  const container = wrapper
+    .dive()
+    .dive()
+    .dive()
+    .dive()
+    .debug();
```
Comment (plzen) [inline src/components/Projects/ProjectList/__tests__/container.spec.js]: Remove `debug()`

File: jest.config.js (hunk 1 above)
Comment (eugene-chr) [reply jest.config.js]: done

File: src/components/Projects/ProjectItem/ProjectModalConfirmDelete/__tests__/container.spec.js (hunk 4 above)
Comment (eugene-chr) [reply src/components/Projects/ProjectItem/ProjectModalConfirmDelete/__tests__/container.spec.js]: done

File: src/components/Projects/ProjectItem/__tests__/container.spec.js (hunk 5 above)
Comment (eugene-chr) [reply src/components/Projects/ProjectItem/__tests__/container.spec.js]: done

File: src/components/Projects/ProjectList/__tests__/container.spec.js (hunk 6 above)
Comment (eugene-chr) [reply src/components/Projects/ProjectList/__tests__/container.spec.js]: done

File: src/components/Projects/ProjectItem/ProjectModalConfirmDelete/__tests__/container.spec.js (hunk 4 above)
Comment (eugene-chr) [reply src/components/Projects/ProjectItem/ProjectModalConfirmDelete/__tests__/container.spec.js]: done

File: src/components/Projects/ProjectItem/__tests__/container.spec.js (hunk 5 above)
Comment (eugene-chr) [reply src/components/Projects/ProjectItem/__tests__/container.spec.js]: done

File: src/components/Projects/ProjectList/__tests__/container.spec.js (hunk 6 above)
Comment (eugene-chr) [reply src/components/Projects/ProjectList/__tests__/container.spec.js]: done

File: src/components/Projects/ProjectForm/__tests__/container.spec.js (hunk 2 above)
Comment (eugene-chr) [reply src/components/Projects/ProjectForm/__tests__/container.spec.js]: done

File: src/components/withAuth.js [hunk 7]
```diff
@@ -14,10 +12,6 @@ const WithAuth = WrappedComponent => {
     }
   }
 
-  const mapStateToProps = state => ({
```
Comment (plzen) [inline src/components/withAuth.js]: Why did you remove this code? is it works?

Comment (eugene-chr) [reply src/components/withAuth.js]: I remove connect to store & we no longer need `mapStateToProps()`. The user now takes from props
